uber/config.py

```diff
@@ -136,6 +136,19 @@ def SUPPORTER_COUNT(self):
                                                 sa.Attendee.amount_paid >= self.SUPPORTER_LEVEL).count()
             return individual_supporters + group_supporters
 
```

Comment:
In general, if we want a property to possibly override a config option, my preference would be to name it the same as the config option.  The way this is written, there's a ``c.SQLALCHEMY_URL`` and a ``c.SQLALCHEMY_FINAL_URL``, but we could just as easily make this property named ``c.SQLALCHEMY_URL`` so as to not burden the programmer with needing to remember which to use.  In this case we'd simply need to replace ``self.SQLALCHEMY_URL`` with ``_config['sqlalchemy_url']`` in the return statement at the end of this function.